### ISSUE TITLE
App.js refactor

### ISSUE THREAD

**Issue by GKStretton** [GKStretton/SE#31]
App.js is out of control (>450 lines so far)
Need to move stuff to different files, e.g. general routes, payment, db, keystone idk
I'm not sure what can be moved where though.

**Comment by Spookyerol** [comment]
I would suggest separating DB, routes, general API and keystone each into their own files, but file size isnt the only thing we need to control, the amount of files we have can just as easily got out of control.

**Comment by nikeshpatel99** [comment]
Yeah I agree, we should minimise the amount of files as well, using large grouping like Erdal said is probably the best idea.
